tests/client/cursor.rs

```diff
@@ -0,0 +1,82 @@
+use bson::Document;
+use bson::Bson::{FloatingPoint, I64};
+use bson::Bson::String as BsonString;
+use mongodb::client::cursor::Cursor;
+use mongodb::client::wire_protocol::flags::{OpInsertFlags, OpQueryFlags};
+use mongodb::client::wire_protocol::operations::Message;
+use std::io::Write;
+use std::net::TcpStream;
+
+#[test]
+fn cursor_features() {
+    match TcpStream::connect("localhost:27017") {
+        Ok(mut stream) => {
+            let docs : Vec<_> = (0..10).map(|i| {
+                let mut doc = Document::new();
+                doc.insert("foo".to_owned(), I64(i));
+
+                doc
+            }).collect();
+
+            let flags = OpInsertFlags::no_flags();
+            let name = "test.test".to_owned();
```

Comment:
Yeah, I guess that's probably a good idea until we have a way of dropping databases.